docs/source/model_design/risk_factors.rst

```diff
@@ -33,6 +33,72 @@ Overview
 What is a risk factor?
 ++++++++++++++++++++++
 
+A risk factor is any attribute whose measure is causally related to the measure 
+of an outcome. Such attributes can range widely, and are categorized by GBD into 
+environmental/occupational, behavioral, and metabolic risks. For example:
+
+  * Environmental/occupational
+
+    * The level of particulate matter in the air of an individual's environment
+
+    * An individual's access or lack thereof to clear water
+
+  * Behavioral
+
+    * An individual's alcohol intake
+
+    * A low level of physical activity
+
+    * A history or lack thereof of childhood abuse0
+
+  * Metabolic
+
+    * The blood pressure of an individual
+
+    * Having or not having a high BMI
+
+For our purposes, the outcomes of these risks will generally be a differential 
+probability of some cause of health loss. However, the outcome of a risk could 
+also be the differential probability of another risk exposure.
+
+  For example, we might say that an individual with the attribute "is a daily 
+  smoker" is :math:`x_1` times more likely to develop lung cancer than if that same 
+  individual had the attribute "has never smoked". We quantify this more 
+  rigorously in our explanation of relative risk (RR) and theoretical minimum risk 
+  level (TMREL).
+
+  We might also say that an individual with the attribute "mother had a BMI of 17 
+  during pregnancy" is :math:`x_2` times more likely to end up with the attribute "low 
+  weight" than if that same child, all other factors held constant, had the 
```

Comment:
To be consistent, I think this should say "low birth weight" instead of just "low weight."

@beatrixkh, Based on our discussion at the documentation session today, I think this example is good, even though "low birth weight" doesn't correspond exactly to a risk in GBD.